### DISCUSSION TITLE
大佬们能出个小程序的插件吗

### DISCUSSION THREAD

**Discussion by liuweiGL** [web-infra-dev/rsbuild#3489] [Ideas]
taro 虽好但后期的文件尺寸是个问题，原生小程序的 npm 简直是折磨人的东西。大佬能出个 @rsbuild/plugin-miniprogram 吗，救救孩子吧

**Comment by sundial-dreams** [comment]
小程序构建究极诡异，入口多，还有分包、分包异步化等一堆东西，rsbuild估计不好整，我们都是自己写小程序的打包工具。。。
